lexicons/app/bsky/actor/defs.json

```diff
@@ -154,6 +155,38 @@
         }
       }
     },
+    "savedFeed": {
+      "type": "object",
+      "required": ["id", "type", "value", "pinned"],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "knownValues": ["feed", "list", "timeline"]
```

Comment:
how actually i see it's `home`

hmmm my notion of those terms is they should be inversed. I think of "timeline" as "deterministic rev-chron timeline" and "home" as "feed in the 0 index of saved feeds"

Reply:
Oh yeah, didn't call that out. We use `home` by default in the app, which refers to the Following feed + fallbacks to discover + integration with merge feed if that's enabled.

We also support `following` as the value though, which is just straight reverse-chron following. I set it up like that so that we could potentially give users the choice between them in the future.

Reply:
err maybe both the `type` and `value` should be "timeline" 🤔

Reply:
Yeah `home` is a legacy term here, not ideal. Our app supports `home` and `following` in addition to feed/list URIs. `timeline` is a parent term for "following" and we can tweak within that. That seem reasonable?

Reply:
ah cool that makes sense 👍 yeah i'm down with it

Reply:
Any benefit to changing our `home` in the app to `following` so that this value can be cleaner?